app/src/main/java/com/lagradost/cloudstream3/utils/AppUtils.kt

```diff
@@ -257,6 +259,26 @@ object AppUtils {
                 )
             }
             afterRepositoryLoadedEvent.invoke(true)
+            downloadAllPluginsDialog(url)
+        }
+    }
+
+    fun Activity.downloadAllPluginsDialog(repositoryUrl: String) {
+        runOnUiThread {
+            val context = this
+            val builder: AlertDialog.Builder = AlertDialog.Builder(this)
+            builder.setTitle(
+                "Download all plugins from this repo?"
```

Comment:
R.string :skull:

Reply:
Probably good to include the repo name if that's available without another request :thinking:

Reply:
Done 🚀